jme3-terrain/src/main/resources/Common/MatDefs/Terrain/AdvancedPBRTerrain.frag

```diff
@@ -485,6 +493,16 @@ void main(){
     
      
     float ndotv = max( dot( normal, viewDir ),0.0);
+    #ifdef SPECULAR_AA
+        float sigma = 0.5f;
+        float kappa = 0.18f;
```

Comment:
The default values are set twice, here and in the j3md.
Also, can you remove the "f" suffix? Since it is unsupported in older specs, thanks